torch/_tensor_docs.py

```diff
@@ -1712,6 +1712,13 @@ def add_docstr_all(method, docstr):
 See :func:`torch.histc`
 """)
 
+add_docstr_all('histogram',
+               r"""
+histogram(input, bins, *, range=[], min=None, max=None, weight=None, density=False) -> (Tensor, Tensor)
```

Comment:
In the other docs `range=None`, not an empty list, and min and max are not documented